### ISSUE TITLE
[help] Inference results on the model myself trained (change the training code with LoRA finetune)

### ISSUE THREAD

**Issue by 281LinChenjian** [hpcaitech/Open-Sora#640]
Thank you for your outstanding work. I processed the data according to the method in the Readme. I modified the training code on my own dataset and used LoRA to fine-tune the model. My model has been fine-tuned. However, the visual video effect on the inference script did not meet my expectations. Is it because I have too few training rounds and have not converged yet? But I think my loss is still relatively low. The following is my loss curve.
![image](https://github.com/user-attachments/assets/c50b04ef-3221-4975-bc88-bf6185c667a9)
 I fine-tuned on the character host video, but a mosaic-like result appeared during inference.

https://github.com/user-attachments/assets/a1a93c3d-c367-46a0-b7e4-b864ceba0ead

I don’t know where I went wrong. Can you help me?

**Comment by 281LinChenjian** [comment]
Do I need to train a corresponding VAE myself? Or is the current VAE enough?

**Comment by 281LinChenjian** [comment]
Can anyone provide help or ideas?
